### PR TITLE
Update based on latest images

### PR DESCRIPTION
**Pull request by pjbull**

### CODE REVIEW

File: templates/c2c.yaml [hunk 1]
```diff
@@ -44,11 +44,14 @@ Resources:
           Image: 113913174193.dkr.ecr.us-east-1.amazonaws.com/c2c-ui:latest
           MemoryReservation: 256
           Essential: true
-          Command:
-            - npm run dev
+          Environment:
+            - Name: NGINX_API_URL
+              Value: http://localhost:5001/api
```
Comment (pjbull) [inline templates/c2c.yaml]: This should be the URL for the API service, however we figure out what that may be.

File: templates/c2c.yaml [hunk 2]
```diff
@@ -44,11 +44,14 @@ Resources:
...
+            - Name: NGINX_HOST
+              Value: 0.0.0.0
```
Comment (pjbull) [inline templates/c2c.yaml]: This should be the hostname or IP, whatever that will be

File: templates/c2c.yaml [hunk 3]
```diff
@@ -63,14 +66,17 @@ Resources:
...
             - Name: SECRET_KEY
-              Value: notverysecret
-            - Name: DEBUG
-              Value: "True"
+              Value: REALLY_A_SECRET_1098590109385NLKAJKF
             - Name: DJANGO_SETTINGS_MODULE
-              Value: config.settings.local
+              Value: config.settings.production
+            - Name: DATABASE_URL
+              Value: postgres://postgres@localhost:5432/c2c
```
Comment (pjbull) [inline templates/c2c.yaml]: This should be the connection string for the postgres container, whatever that ends up being

File: templates/c2c.yaml (hunk 1 above)
Comment (techgnosis) [reply templates/c2c.yaml]: The ui container is linked with the api container so the host name is 'c2c-api'

File: templates/c2c.yaml (hunk 3 above)
Comment (techgnosis) [reply templates/c2c.yaml]: Hostname for the postgres contaner is 'c2c-postgres'

**Pull request merged**